src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -386,53 +386,34 @@ def build_udtf_apply(
     build_fn_apply_args(ast, *args, **kwargs)
 
 
-def build_session_table_fn_apply(
+def build_call_table_function_apply(
     ast: proto.Expr,
```

Comment:
is there a reason why the alternate syntax `["database", "schema", "entity"]` is not supported anymore? It seems prevalent throughout snowpark, so maybe we should keep supporting it here?

Reply:
nit: I actually liked the `session` being part of the function name, it makes it clear what the call origin is. Unless this is a generalization...

Reply:
This is a misleading diff: `build_session_table_fn_apply` is gone altogether, and `build_call_table_function_apply` is a new thing. They just happen to share similar surrounding text.

Reply:
Oh, and with respect to `functions.call_table_function`, I think there's no technical reason why it couldn't take a `database.schema.function` name passed as a list, and the AST supports this internally, but the API itself is inconsistent with the rest of Snowpark. I won't try to fix that in this change.

Reply:
Done.